cdc/entry/mounter.go

```diff
@@ -126,60 +108,35 @@ func (m *mounterImpl) Run(ctx context.Context) error {
 		select {
 		// ctx.Done returns when parent ctx done or error occurs in errg.
 		case <-ctx.Done():
-			return errg.Wait()
+			return ctx.Err()
 		case <-timer.C:
-			chSize := 0
-			for _, ch := range m.rawRowChangedChs {
-				chSize += len(ch)
-			}
-			metricMounterInputChanSize.Set(float64(chSize))
 			timer.Reset(flushMetricsInterval)
 		}
 	}
 }
 
-func (m *mounterImpl) codecWorker(ctx context.Context, index int) error {
-	changefeedID := util.ChangefeedIDFromCtx(ctx)
-	metricMountDuration := mountDuration.WithLabelValues(changefeedID)
-	metricTotalRows := totalRowsCountGauge.WithLabelValues(changefeedID)
+func (m *mounterImpl) DecodeEvent(ctx context.Context, pEvent *model.PolymorphicEvent) error {
+	m.metricMounterInputChanSize.Inc()
```

Comment:
Let's remove it if there is no channel at all.

Reply:
done